Formula/step.rb

```diff
@@ -34,13 +28,19 @@ def install
       contents = Dir["{*,.git,.gitignore}"]
       (buildpath/"src/github.com/smallstep/certificates").install contents
       cd "#{buildpath}/src/github.com/smallstep/certificates" do
+        ENV["VERSION"] = "v0.8.4"
```

Comment:
It would also be preferred if we didn't have to update this at every release of the resource. I see this in the code
```
# Version flags to embed in the binaries
VERSION ?= $(shell [ -d .git ] && git describe --tags --always --dirty="-dev")
VERSION := $(shell echo $(VERSION) | sed 's/^v//')
```
so it might be better to use a git checkout instead of downloading the file so the version can be extracted from git instead of having to be inserted in the formula.

Reply:
You're totally right. I replied above, but essentially I agree that git should be doing the work for me here.